### PR TITLE
Fixes edit feedback issue 

### LINKED ISSUE #2948: Edit feedback issue
**Actual Behaviour**

While editing the feedback of a skill the previous feedback does not appear in the textbox.

**Expected Behaviour**

Previous feedback must appear when a user tries to edit feedback.

**Screenshots of the issue**

ezgif com-video-to-gif (2)

**Would you like to work on the issue?**
 
Yes, I would like to work on this issue

### PR DESCRIPTION
**Pull request by Dilshaad21**
Fixes #2948 

Changes: 
**In SkillFeedbackCard.js**
-A `componentDidMount` life cycle hook is used to intialise the `newFeedbackValue` to the previous feedback submitted by the user.

**In index.js**
-`EditFeedbackDialog` component was imported from `EditFeedbackDialog.js` instead of `EditFeedback`.
Demo Link: https://pr-2956-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/cms/SkillFeedbackCard/SkillFeedbackCard.js [hunk 1]
```diff
@@ -63,14 +63,14 @@ class SkillFeedbackCard extends Component {
...
-  handleEditOpen = () => {
+  handleEditOpen = previousFeedback => {
     this.handleMenuClose();
     this.props.actions.openModal({
       modalType: 'editFeedback',
       handleConfirm: this.postFeedback,
       handleClose: this.props.actions.closeModal,
       errorText: this.state.errorText,
-      feedback: this.state.newFeedbackValue,
+      feedback: previousFeedback,
```
Comment (akshatnitd) [inline src/components/cms/SkillFeedbackCard/SkillFeedbackCard.js]: This approach won't work. You need to initialise the `newFeedbackValue` state variable with `data.feedback` to achieve the desired behaviour.

Comment (Dilshaad21) [reply src/components/cms/SkillFeedbackCard/SkillFeedbackCard.js]: @akshatnitd once user has posted a feedback, then `newFeedbackValue` is required only when user tries to edit it. So why not I intialise it when user clicks the edit button.

**Comment by Dilshaad21** [comment]
@akshatnitd Please take a look. Is this approach fine?

**Comment by Dilshaad21** [comment]
@akshatnitd do you mean passing the `data.feedback` to `handleEditOpen()` and then pass it to modal replacing `this.state.newFeedbackValue`.

**Comment by akshatnitd** [comment]
@Dilshaad21 Yes. I think I understood the implementation differently that time. Sorry for that. Please make the same changes as before.

**Comment by Dilshaad21** [comment]
@akshatnitd the way I did it using `componentDidMount` is able to initialise the local variable `newFeedbackValue`. Or should I go for some other approach.

**Comment by akshatnitd** [comment]
> @akshatnitd Sir do you mean passing the `data.feedback` to `handleEditOpen()` and then pass it to modal replacing `this.state.newFeedbackValue`. This is the same way I did at the beginning.[#2956 (comment)](https://github.com/fossasia/susi.ai/pull/2956#discussion_r336586085)
> 
> Are you saying to implement it this way:
> `handleEditOpen = previousFeedback => {`
> ` this.handleMenuClose();`
> ` this.setState({ newFeedbackValue: previousFeedback }, () => {`
> ` this.props.actions.openModal({`
> ` modalType: 'editFeedback',`
> ` handleConfirm: this.postFeedback,`
> ` handleClose: this.props.actions.closeModal,`
> ` errorText: this.state.errorText,`
> ` feedback: this.state.newFeedbackValue,`
> ` handleEditFeedback: this.editFeedback,`
> ` });`
> ` });`
> ` };`

@Dilshaad21 Use this approach.

**Comment by Dilshaad21** [comment]
@akshatnitd PTAL.

**Pull request merged**